apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/NtpTime.java

```diff
@@ -58,9 +62,9 @@ public NtpTime() {}
 
     @PostConstruct
     public void start() {
+        setUpClient();
         Runnable timeUpdate = this::setTimeDrift;
         ThreadPool.scheduleThread("NTP Update", timeUpdate, REFRESH_FREQUENCY, TimeUnit.SECONDS);
-        setUpClient();
     }
 
     private void setUpClient() {
```

Comment:
I would recommend catch here two errors (or IO)

... catch (SocketException | UnknownHostException e) {
OR
... catch (IOException  e) { // that replaces both listed above

Reply:
Why we should catch such exceptions?